.gitignore

```diff
@@ -1,2 +1,5 @@
 *.o
-src/hello
\ No newline at end of file
+src/hello
+bomsh
```

Comment:
bombash

Reply:
oh, actually, since the three files are put into ./out dir, so we can ignore the whole ./out dir?

Reply:
The out dir exists only in the container... not on the host.